stripe/src/test/java/com/stripe/android/view/ExpiryDateEditTextTest.kt

```diff
@@ -257,6 +267,17 @@ class ExpiryDateEditTextTest {
             .isEqualTo(0)
     }
 
+    @Test
+    fun inputCompleteDate_whenMonthInvalid_showsInvalidMonth() {
+        expiryDateEditText.append("15")
+        expiryDateEditText.append("50")
+
+        assertThat(expiryDateEditText.shouldShowError)
+            .isTrue()
+        assertThat(expiryDateEditText.errorMessage)
+            .isEqualTo(context.getString(R.string.invalid_expiry_month))
```

Comment:
I'd slightly prefer to use the actual string instead of the string resource because it makes the test more readable, but this is fine.